src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,185 @@
+import uniqId from 'uniqid';
+import {Button} from '../Button/Button';
+import './AddUserForm.scss';
+import ImgWarning from './../../assets/icon-attention-png-clip-art.png'
+
+const CN = 'user-form';
+
+class AddUserForm extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentError: '',
+            inputErrorName: '',
+            inputErrorLastName: '',
+            errorInputName: '',
+            errorInputLastName: ''
+        };
+    }
+
+    // todo: добавить state, в него добавить пропертю, которая будет отображать есть ли ошибка или нет.
+    //  допустим это будет строка currentError
+    //  по умолчанию она будет пустой
+    //  если произошла ошибка - в эту пропертю запишется строка, какая именно ошибка произошла
+
+    userNameRef = createRef();
+    userLastNameRef = createRef();
+
+    onInputChange = (e) => {
+
+        if (e.target.value) {
+            this.setState({
+                currentError: ''
+            })
+        }
+
+        if (this.userNameRef.current.value) {
+            this.setState({
+                inputErrorName: '',
+                errorInputName: ''
+            })
+        }
+
+        if (this.userLastNameRef.current.value) {
+            this.setState({
+                inputErrorLastName: '',
+                errorInputLastName: ''
+            })
+        }
+
+        // todo: если в стейте была записана какая-то ошибка - зачищаем ее при изменении инпута
+        //  поскольку в ситуации если пользователь ничего - не ввел - увидел ошибку - понял, что не так - начал вводить данные
+        //  было бы хорошо не пугать его висящим сообщением, что все таки что-то не так
+    };
+
+    onSubmit = (e) => {
+        e.preventDefault();
+
+        if (this.userNameRef.current.value.trim() === '' && this.userLastNameRef.current.value.trim() === '') {
+            this.setState({
+                currentError:
+                    <div className='warning'>
+                        <div>
+                            <h2>WARNING!!! WRITE SOMETHING!!!</h2>
+                            <img src={ImgWarning} alt="Warning"/>
+                        </div>
+                    </div>,
+            });
+        }
+
+        if (!this.userNameRef.current.value.trim()) {
+
+            this.setState({
+                inputErrorName: 'second-form-control',
+                errorInputName: 'This input is empty! Write something!'
+
+            })
+        }
+
+        if (!this.userLastNameRef.current.value.trim()) {
+
+            this.setState({
+                inputErrorLastName: 'second-form-control',
+                errorInputLastName: 'This input is empty! Write something!'
+            })
+        }
+
+        // Todo: добавить здесь проверку, если пользователь НЕ ВВЕЛ в инпуты ничего - не создаем пост
+        //  и обновляем currentError в стейте, кладем сообщение, что пошло не так
+        //  если пользователь ввел данные - создаем постзадачу
```

Comment:
I guess you still can add user with empty name and last name
missing return statement in case of errors